### ISSUE TITLE
Logo alt text

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2498]
## Source

Pacific ADA

## Describe the bug

The logo reads "Home"   This is good to indicate where it takes a user if selected, but there is no alt text to tell a user using a screen reader what the logo looks like.  	

## Recommendations

Add alt text field in site identity for logo, default text to be 'Logo shows the text for [sitename].

Add information to the alt text to say "Pacific ADA Center".  So for example:

`alt="Home  Logo shows the text Pacific ADA Center"`

## Screenshots

![home logo](https://github.com/proudcity/wp-proudcity/assets/1569463/100301ed-0875-4609-8afb-8ea1a59bb274)

**Comment by curtismchale** [comment]
@lukefretwell we can add the ability for our customers to add this content in the customizer.

![alt-text-customizer-logo](https://github.com/proudcity/wp-proudcity/assets/74561/8d0adcf3-73e7-4799-bd19-f575d5d2d6ee)

**Comment by lukefretwell** [comment]
@curtismchale that would be fantastic.
